packages/flutter/lib/src/foundation/basic_types.dart

```diff
@@ -34,25 +34,48 @@ class BitField<T extends dynamic> {
   static const int _kSMIBits = 63; // see https://www.dartlang.org/articles/numeric-computation/#smis-and-mints
   static const int _kAllZeros = 0;
   static const int _kAllOnes = 0x7FFFFFFFFFFFFFFF; // 2^(_kSMIBits+1)-1
```

Comment:
Do you want to file a bug about that?  It doesn't seem like something I should fix in this patch.